### ISSUE TITLE
Bug? Flying on Server with no Gamemode

### ISSUE THREAD

**Issue by BlawoHD** [TesseractTeam/Tesseract#149]
#### Problem description

When I joine on my server, I can fly without Gamemode & I have no Fly plugin on it. Is this a bug or is it due to a plugin?
#### OS and versions

* Tesseract: Tesseract_1.0.0dev_2017-01-23_06-15-03
* PHP: 7.0.9
* Server OS:
* Game version: MCPE v1.0.2.1
* Server host: http://server.pro
<br><br>
#### Crashdump, backtrace, memory dumps, plugins list or other files

```
```

**Comment by ankitmaharjan252** [comment]
What is the gamemode for you when you join?

**Comment by remotevase** [comment]
This is absolutely unconfirmed, as I have been running Tesseract for months and a server for 3 years, and I have never experience this, except for two times. 
1. When I installed a plugin which had a piece of code which was unknown to me... I modified the plugin then it worked
2. When I accidentally set default GM to adventure not survival so I could fly and it **looked** like survival but it wasn't

Solutions: Try without plugin or double check the gamemode when you join, like @ankitmaharjan252 suggested. Whatever this is, there is a 98% chance that this issue is not the fault of our code and that it is actually just a personal issue from the settings.
